cli.ts

```diff
@@ -129,6 +129,31 @@ const commonCLI = (env:EnvVars, args:DenoArgs, i18n: i18n) =>
             forkCatch (console.error) (console.error) (console.log)
         )
     )
+    .command(
+        'scaffold [scaffoldUrl]',
+        i18n.__('scaffoldDesc'),
+        (yargs: Arguments) => {
+            yargs
+                .positional('scaffoldUrl', {
+                    describe: i18n.__('scaffoldUrlDesc'),
+                    type: 'string',
+                    default: 'https://github.com/ecadlabs/taqueria-scaffold-quickstart.git',
+                })
+                .positional('scaffoldProjectDir', {
+                    alias: 'scaffoldProjectDir',
+                    type: 'string',
+                    describe: i18n.__('scaffoldProjectDirDesc'),
+                    default: './taqueria-quickstart',
+                })
+        },
+        (args: RawInitArgs) => pipe(
+            sanitizeScaffoldArgs(args), 
+            ({scaffoldUrl, scaffoldProjectDir, configDir, maxConcurrency, quickstart}: SanitizedScaffoldInitArgs) => {
+                return scaffoldProject(scaffoldUrl, scaffoldProjectDir, configDir, i18n, maxConcurrency, quickstart)
+            },
+            fork (console.error) (console.log)
```

Comment:
This is new - in one of my latest branches, I switched all uses of `fork` to `forkCatch`. You can read about the differences here:
https://github.com/fluture-js/Fluture#fork

TLTR excerpt from their docs:
> If an exception was encountered during the computation, it will be re-thrown by fork and likely not be catchable. You can handle it using process.on('uncaughtException') in Node, or use [forkCatch](https://github.com/fluture-js/Fluture#forkcatch).